en/docs/references/ballerina-expression-syntax.md

```diff
@@ -492,3 +556,31 @@ To convert the `theRevenantJson` variable of type `json` to the record `Movie`,
 ```ballerina
 check theRevenantJson.cloneWithType(Movie);
 ```
+
+## XML
+
+XML is a textual format for representing a string or an XML object. The following topics describe these values.
```

Comment:
We don't use the term XML object. Shall we revisit this entire section?

Relevant section in the spec - https://ballerina.io/ballerina-spec/spec.html#XML.

Reply:
Done